### PR TITLE
Add of big-nosed-bugs to story in 'english' directory

### PR DESCRIPTION
**Pull request by gpickard**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1820)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, please review this

File: english/big-nosed-bugs/big-nosed-bugs.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+You have arrived in a land full of ... well Big Nosed Bugs.
```
Comment (casey-collab) [inline english/big-nosed-bugs/big-nosed-bugs.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/big-nosed-bugs/big-nosed-bugs.md (hunk 1 above)
Comment (gpickard) [inline english/big-nosed-bugs/big-nosed-bugs.md]: Ah you are right! I will do that, thank you! 
 
 
 
 
-------Original Message-------
 
From: Casey Collaborator
Date: 7/8/2015 3:55:52 PM
To: udacity/create-your-own-adventure
Cc: gpickard
Subject: Re: [create-your-own-adventure] Add of big-nosed-bugs to story in 
english' directory (#1820)
 
In english/big-nosed-bugs/big-nosed-bugs.md:
> @@ -0,0 +1,8 @@
> +You have arrived in a land full of ... well Big Nosed Bugs.

It looks like this page is not reachable from the rest of the story. That is
 if a reader starts in language.md and keeps clicking links, they will never
find this page. You can fix this by finding a good place in the existing
story to add an option with a link that leads to this page.

**Comment by cbuckey-uda** [comment]
@gpickard I have not seen any updates yet. Did you make the updates and forget to push, or have you not had time yet?

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**